modules/misc/qt.nix

```diff
@@ -21,18 +21,67 @@ in
         default = false;
         description = ''
           Whether Qt 4 and 5 should be set up to use the GTK theme
-          settings.
```

Comment:
You can remove the `useGtkTheme` option, the assertion, and the warning. Instead add something like
```
imports = [
  (mkChangedOptionModule
    [ "qt" "useGtkTheme" ]
    [ "qt" "platformTheme" ]
    (config:
      if getAttrFromPath [ "qt" "useGtkTheme" ] config
      then "gtk"
      else null))
]
```
(untested but I think it should be basically right 🙂)

Reply:
Changed, thanks!